browser/js/common/controller/user/user.controller.js

```diff
@@ -0,0 +1,7 @@
+app.controller('userDetailController', function ($scope, $log, $state, loggedInUser) {
+	if (loggedInUser) {
+		$scope.user = loggedInUser;
+	} else {
+		$state.go('notLoggedIn');
+	}
+});
```

Comment:
Just a controller so that the personal user page can use the resolved 'loggedInUser'